tests/ceph_volume/test_ceph_volume_redeploy.py

```diff
@@ -70,17 +70,40 @@ def run(ceph_cluster, **kw):
     size = device_spec.get("data_devices", {}).get("size", {}).replace("GB", "")
     _add_node_volumes(nodes=osd_nodes, config=config, size=size, count=4)
 
+    # Get required host from config specs
+    nodes = config.get("spec", {}).get("placement", {}).get("hosts")
+    host = [ceph_cluster.get_nodes()[int(node[-1])].hostname for node in nodes]
+
+    # Create a spec file specific to hostname
+    specs = config.get("spec", {})
+    specs["placement"]["hosts"] = host
+
+    # Refresh ceph orch devices
+    c = {"refresh": True}
+    devices = CephAdm(installer).ceph.orch.device.ls(**c)
```

Comment:
Can be changed to :
`if not CephAdm(installer).ceph.orch.device.ls(**c):`

Reply:
This we can do it.